src/pages/[category]/[slug].tsx

```diff
@@ -14,6 +14,7 @@ import {
 type PageProps = {
   data: {
     component?: string
```

Comment:
Should the component be mandatory?

Reply:
Nope, because some pages don't have a component to render a props table for